consts.js

```diff
@@ -1,9 +1,11 @@
-const EXPIRES_INVALID_MESSAGE = "\"Expires\" param must be an integer";
-const CLUSTER_REQUIRED_MESSAGE = "Cluster name is required";
-const MISSING_OR_INCORRECT_CREDENTIALS_ERROR = "Missing or incorrect credentials - please select valid access and secret keys first";
+const MISSING_OR_INCORRECT_CREDENTIALS_MESSAGE = "Missing or incorrect credentials - please select valid access and secret keys first";
```

Comment:
This message doesn't seem to be used anywehere.

Reply:
It is used in [autocomplete.js](https://github.com/Kaholo/kaholo-plugin-aws-eks/pull/5/files#diff-8d2a05e5cc6543b2285d1de69039fa5a6c29757286c71d9ff81f68aafab30886) at lines [58](https://github.com/Kaholo/kaholo-plugin-aws-eks/pull/5/files#diff-8d2a05e5cc6543b2285d1de69039fa5a6c29757286c71d9ff81f68aafab30886R58) and [49](https://github.com/Kaholo/kaholo-plugin-aws-eks/pull/5/files#diff-8d2a05e5cc6543b2285d1de69039fa5a6c29757286c71d9ff81f68aafab30886R49).

Reply:
It's `MISSING_OR_INCORRECT_CREDENTIALS_ERROR`, running this code would throw type errors.
Please rename it to `MISSING_OR_INCORRECT_CREDENTIALS_ERROR_MESSAGE` to emphasize it's function.